FIDO2_make_authenticator_data_req.md

```diff
@@ -0,0 +1,22 @@
+## FIDO2 Make Authenticator Data Request
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Ditto